src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/SmoModel/SmoChildFactoryBase.cs

```diff
@@ -1,133 +1,169 @@
-using Microsoft.SqlServer.Management.Smo;
-using Microsoft.SqlTools.ServiceLayer.ObjectExplorer.Nodes;
-using Microsoft.SqlTools.ServiceLayer.Utility;
+using System.Linq;
+using System.Reflection;
+using Microsoft.SqlServer.Management.Smo;
+using Microsoft.SqlTools.ServiceLayer.ObjectExplorer.Nodes;
 using Microsoft.SqlTools.Utility;
 
-namespace Microsoft.SqlTools.ServiceLayer.ObjectExplorer.SmoModel
-{
-    public class SmoChildFactoryBase : ChildFactory
-    {
-        public override IEnumerable<string> ApplicableParents()
-        {
-            return null;
-        }
-        
-        public override IEnumerable<TreeNode> Expand(TreeNode parent)
-        {
-            //parent.BeginChildrenInit();
-            try
-            {
-                List<TreeNode> allChildren = new List<TreeNode>();
-                OnExpandPopulateFolders(allChildren, parent);
-                RemoveFoldersFromInvalidSqlServerVersions(allChildren, parent);
-                OnExpandPopulateNonFolders(allChildren, parent);
-                OnBeginAsyncOperations(parent);
-                return allChildren;
-            }
-            finally
-            {
-                //parent.EndChildrenInit();
-            }
-        }
-        
-        /// <summary>
-        /// Populates any folders for a given parent node 
-        /// </summary>
-        /// <param name="allChildren">List to which nodes should be added</param>
-        /// <param name="parent">Parent the nodes are being added to</param>
-        protected virtual void OnExpandPopulateFolders(IList<TreeNode> allChildren, TreeNode parent)
-        {
-        }
-
-        /// <summary>
-        /// Populates any non-folder nodes such as specific items in the tree.
-        /// </summary>
-        /// <param name="allChildren">List to which nodes should be added</param>
-        /// <param name="parent">Parent the nodes are being added to</param>
-        protected virtual void OnExpandPopulateNonFolders(IList<TreeNode> allChildren, TreeNode parent)
-        {
-            if (ChildQuerierTypes == null)
-            {
-                // This node does not support non-folder children
-                return;
-            }
-            SmoQueryContext context = parent.GetContextAs<SmoQueryContext>();
-            Validate.IsNotNull(nameof(context), context);
-            IEnumerable<SmoQuerier> queriers = context.ServiceProvider.GetServices<SmoQuerier>(q => IsCompatibleQuerier(q));
-            foreach (var querier in queriers)
-            {
-                foreach(var smoObject in querier.Query(context))
-                {
+namespace Microsoft.SqlTools.ServiceLayer.ObjectExplorer.SmoModel
+{
+    public class SmoChildFactoryBase : ChildFactory
+    {
+        public override IEnumerable<string> ApplicableParents()
+        {
+            return null;
+        }
+        
+        public override IEnumerable<TreeNode> Expand(TreeNode parent)
+        {
+            //parent.BeginChildrenInit();

```

Comment:
We should probably remove the commented out parts. My memory is it was SSDT presentation-related so isn't vital for us.